tsdb/index/tsi1/index.go

```diff
@@ -802,21 +802,11 @@ func (i *Index) compactionGroups(fileSet FileSet) [][]*IndexFile {
 
 		// Skip over log files. They compact themselves.
 		if !ok {
-			if isCompactableGroup(group, i.CompactionFactor) {
-				group, groups = nil, append(groups, group)
-			} else {
-				group = nil
-			}
 			continue
 		}
```

Comment:
This section is needed because there can be `tsi` files before it reaches a log file. Those files need to be grouped in a compaction group.